tests/integration/targets/test_upload_file_to_s3/runme.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Just curious if there's a reason this is run as a bash script instead of an ansible task as in the role tests?

Reply:
There is! Found it myself when working on the upload to GCP playbook :)